core/src/main/java/jenkins/mvn/GlobalSettingsProvider.java

```diff
@@ -21,13 +26,45 @@
 public abstract class GlobalSettingsProvider extends AbstractDescribableImpl<GlobalSettingsProvider> implements ExtensionPoint {
 
     /**
+     * <p>
```

Comment:
It may break the summary line in Javadoc. I would keep the first entry without paragraph tags